README.md

```diff
@@ -12,12 +12,17 @@ Here are some sample use cases of GPTScript:
 
 
 ### Getting started
-MacOS and Linux:
+MacOS and Linux (Homebrew):
 ```
 brew install gptscript-ai/tap/gptscript 
 gptscript github.com/gptscript-ai/llm-basics-demo
 ```
 
+MacOS and Linux (install.sh):
+```
+curl https://get.gptscript.ai/install.sh | sh
+```
+
```

Comment:
IIRC script is also set up to pull windows releases, but I've never verified that.